libs/api/domains/communications/README.md

```
@@ -5,3 +5,20 @@ Endpoint to route information to service desk
 ## Running unit tests
 
 Run `yarn test api-domains-communications` to execute the unit tests via [Jest](https://jestjs.io).
+
+## Injecting form field values into the email subject
+
+The `emailSubject` field on a `form` entry in Contentful can reference the values that the user
+entered by wrapping a form field reference in square brackets:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)